content/docs/command-reference/fetch.md

```diff
@@ -17,22 +18,16 @@ positional arguments:
 
 ## Description
 
-Downloads DVC-tracked files from remote storage into the cache of the project
-(without placing them in the <abbr>workspace</abbr>, like `dvc pull` would).
-This makes them available for linking (or copying) into the workspace (refer to
-[`dvc config cache.type`](/doc/command-reference/config#cache)).
+`dvc fetch` downloads tracked data from remote storage into the
```

Comment:
It's better to avoid starting command reference `dvc fetch` with `dvc fetch`

Reply:
Agree, starting at "Downloads" directly.